sycl/include/sycl/ext/oneapi/reduction.hpp

```diff
@@ -1034,16 +1034,29 @@ struct get_reduction_aux_kernel_name_t<sycl::detail::auto_name, Type, B1, B2,
 };
 
 /// Called in device code. This function iterates through the index space
-/// \p Range using stride equal to the global range specified in \p NdId,
+/// by assigning contiguous chunks to each work-group, then iterating
+/// through each chunk using a stride equal to the work-group's local range,
 /// which gives much better performance than using stride equal to 1.
 /// For each of the index the given \p F function/functor is called and
 /// the reduction value hold in \p Reducer is accumulated in those calls.
 template <typename KernelFunc, int Dims, typename ReducerT>
-void reductionLoop(const range<Dims> &Range, ReducerT &Reducer,
+void reductionLoop(const range<Dims> &Range, const size_t PerGroup,
+                   const size_t /*Remainder*/, ReducerT &Reducer,
```

Comment:
John, this argument 'Reminder' is unused, then why passing it to here? Do you have some plan for it for later?